packages/service-library/src/servicelib/long_running_tasks/http_endpoint_responses.py

```diff
@@ -25,14 +30,25 @@ async def get_task_result(
     tasks_manager: TasksManager, task_context: TaskContext | None, task_id: TaskId
 ) -> Any:
     try:
-        return tasks_manager.get_task_result(
-            task_id=task_id, with_task_context=task_context
+        task_result = tasks_manager.get_task_result(
+            task_id, with_task_context=task_context
+        )
+        await tasks_manager.remove_task(
+            task_id, with_task_context=task_context, reraise_errors=False
         )
-    finally:
-        # the task is always removed even if an error occurs
+        return task_result
+    except (TaskNotFoundError, TaskNotCompletedError):
+        raise
+    except Exception as exc:
+        # the task raised an exception
+        formatted_traceback = "".join(traceback.format_exception(exc))
+        _logger.info("Task '%s' raised an exception: %s", task_id, formatted_traceback)
```

Comment:
Same as above:

- logging exceptions that you re-raise in a library i do not find it very useful
- use `create_troubleshotting_log_kwargs` to log exceptions regardless of the log level... 
   -  if you do that, you do not need to do this funny traceback ... it is there automatically

Reply:
Yes, I went ahead and used your suggestion